mw-webapp/src/styles/_variables.scss

```diff
@@ -12,4 +12,6 @@ $link-active: rgb(255, 149, 0);
 $header-background: rgb(85, 122, 149);
 
 // Title
-$title-padding: 10px;
\ No newline at end of file
+$title-color: rgb(246, 246, 150);
```

Comment:
looks like all these 3 variable should be used only together for titles, right? And they could not be used separately... time to use mixins!

Reply:
to be honest, I don't understand where this mixin will be reused, except in the title. 
But I added this